contracts/strategies/allocation/donation-voting-passport-offchain-calc/README.md

```diff
@@ -0,0 +1,57 @@
+    - **Customizable Settings**
+        - pool admin can require recipients to have Allo registry identity in order to apply
+            - if pool admin doesn't require Allo identity and the recipient doesn't have one, then `msg.sender` is used as recipient id
+        - pool admin can include optional and required questions in an application, stored in metadata
+            - recipient must answer questions as specified by pool admin
+        - pool admin can require the recipient to have verified credentials for Twitter or Github
+            - recipient must have those credentials on their recipient id
+- `reviewApplication` - function for pool admins to manually accept applications into the round
+    - If a pool admin accepts the application, the recipient status is updated to `Approved`
+    - If a pool admin rejects the application, the recipient status is updated to `Rejected`
+
+### Voter eligibility logic
+In this strategy, the pool admin can use Passport to signal whether the voter should be considered eligible. 
+
+- the pool admin should be able to configure whether they want to use Passport for voter eligibility signalling
+    - If they choose to configure, all voters will be marked as `eligible` or `ineligible` depending on the Passport score
+    - If they do not configure, all voters will be marked as `eligible`
+
+### Voting logic
+In this strategy, voters submit a vote by donating tokens to recipients. The donations are held in the protocol until the round ends, then the pool admin is able to release those funds. Any voter can submit a donation, but only eligible voters (as measured by Passport) will receive matching in the off-chain calculation.  
+
+- `vote` - function for voters to submit donations to recipients
+    - voters can submit donations regardless of their eligibility status
+    - **Customizable Settings** 
+        - pool admin can create an allowlist of tokens that will be accepted for donations
+            - If a voter tries to submit unapproved tokens the transaction should revert
+
+### Allocation shape
+In this strategy, the funding pool is technically distributed proportionally to votes but everything is calculated offchain. The pool admin will need to upload a final allocation to the contract, which should be the amount of the pool token that each recipient should receive. 
+
+- `uploadAllocation` - function for pool admins to upload final allocation
+
+### Distribution logic
+In this strategy, the pool admins are able to push pool funds to recipients. They are also able to release the donation funds.
+
+- `distribute` - function for pool admin to distribute pool funds to recipients, as indicated by the allocation added with `uploadAllocation`
+    - If there is no allocation uploaded, then this should revert
+    - The pool admin should have the ability to distribute the pool in batches
+- `releaseDonations` - function for pool admin to release donation funds to recipients
```

Comment:
I think `releaseDonations` is not needed, or should be rephrased into a more generic name like `claim` or `withdraw` which can be used in different strategies. 
Also it shouldn't be restricted, since an inactive pool admin would cause all funds to be lost.

Reply:
Good call — I changed this to automatically distribute the donations when the allocation window ends